prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,12 @@
+const languages = [
+  'html', 'css', 'javascript', 'react', 'node', 'postgres'
+];
+console.log(languages);
+var firstElement = languages[0]
+console.log('The first language is', firstElement);
+var thirdElement = languages[2]
+console.log('The third language is', thirdElement);
+var length = languages.length
+console.log('The length of the array is', languages.length)
+var lastElement = languages[5]
```

Comment:
Try using your `length` variable to calculate the last index of the array. It's not always possible to know just how long an array is.